inventory/group_vars/all/vars.yml

```diff
@@ -115,6 +115,7 @@ timezone: America/New_York
 ## replication playbook settings
 timestamp: "{{ ansible_date_time.date }}"
 deploy_user: "conan"
+deploy_user_uid: 1002
```

Comment:
This seems a bit fragile. If you are comfortable with it, I do not object, but if we could find a way to make things work without mandating a UID, the system might be more resilient.

Reply:
I agree that it's weird. I wouldn't normally go this route, but TigerData permissions _require_ uid/gid mapping for the permissions configuration. Usually `conan` gets this uid anyway, but it depends on whether or not the PUL `deploy` user is created on our VMs. Our recent slack conversation helped me recognize that at least I could enforce the uid to be consistent since it is currently required for TigerData.

Reply:
I thought I put a comment somewhere about this - I'll add a note with this variable so we'll at least know why it's there